stimuli-ci-generation/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM rocker/tidyverse:3.6.3
```

Comment:
Was this the base file that we're NOT supposed to use because this took too long to build?

Reply:
rocker (r + docker) is a docker image containing all the r packages pre-installed. This particular image has tidyverse- a collection of popular data science libs, installed in it. These packages might be used to generate stimuli using R scripts.

Reply:
sorry I meant to highlight the whole file ~~~ as in was this Dockerfile the one we're not supposed to use?

Reply:
Oh no problem, I thought you asked about the base image.

Reply:
Yes, @jaynilpatel  is correct.